sdk_container/src/third_party/coreos-overlay/sys-firmware/edk2-aarch64/edk2-aarch64-20220221.ebuild

```diff
@@ -0,0 +1,23 @@
+# Copyright (c) 2024 The Flatcar Maintainers.
```

Comment:
Just a question, should we also update edk2 for amd64?

Reply:
Dunno if this question was answered. Does it block the PR?

Reply:
It's not a blocker, but ideally we would have them in sync.